pom.xml

```diff
@@ -1679,28 +1681,20 @@
               <algorithms>
                 <algorithm>SHA-512</algorithm>
               </algorithms>
-              <failOnError>false</failOnError>
+              <csvSummary>false</csvSummary>
             </configuration>
           </plugin>
-          <!-- override the parent assembly execution to customize the assembly
-            descriptor and final name -->
+
+          <!-- override the parent assembly execution to customize the assembly final name -->
           <plugin>
             <artifactId>maven-assembly-plugin</artifactId>
+            <version>3.1.0</version>
             <executions>
               <execution>
                 <id>source-release-assembly</id>
-                <phase>package</phase>
-                <goals>
-                  <goal>single</goal>
-                </goals>
                 <configuration>
-                  <runOnlyAtExecutionRoot>true</runOnlyAtExecutionRoot>
                   <appendAssemblyId>false</appendAssemblyId>
-                  <descriptors>
-                    <descriptor>src/main/resources/assemblies/source-assembly.xml</descriptor>
```

Comment:
How are we making sure that excludes defined in source-assembly.xml are still taken care of ?

Reply:
I have manually checked that all files are the same which obtained in the result of `mvn clean install -DskipTests -Papache-release`.
Looks like there are no differences, except those mentioned in Laurent's comment in the PR #449